src/components/chat/ChatList.js

```diff
@@ -0,0 +1,59 @@
+
+import React, { Component } from 'react';
+import { Link } from "react-router-dom"
+import "./chatList.css"
+import ChatManager from "../../modules/ChatManager"
+
+export default class ChatList extends Component {
+    render() {
+        console.log`messages: ${this.props.messages}`
+        return (
+            <section className="list">
+                <h3 className="list-title">Messages</h3>
+                <button type="button"
+                    className="btn btn-success"
+                    onClick={() => {
+                        this.props.history.push("/messages/new")
+                    }
+                    }>
+                    Add Message
+               </button>
+
+                {this.props.messages.map(message =>
+                    <div key={message.id} className="card ">
+                        <div className="card-body">
+                            <h4 className="card-title">
+                                <div className="text"> {message.text} </div>
+                                <div className="userName"> {`${message.user.username}`}</div>
+                                <div className="timestamp"> {`${message.timestamp}`}</div>
```

Comment:
for some reason not all the timestamps are rendering on the dash in the same format